mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -48,6 +48,55 @@ export class CompareRunView extends Component {
       },
     );
     Utils.updatePageTitle(pageTitle);
+    this.adjustTableColumnWidth(this.props.runInfos.length);
+    window.addEventListener(
+      'resize',
+      (e) => this.adjustTableColumnWidth(this.props.runInfos.length),
+      true,
+    );
+
+    function onTableBlockScrollHanlder(e) {
+      const blocks = document.querySelectorAll('.compare-table .table-block');
+      for (let index = 0; index < blocks.length; ++index) {
+        const block = blocks[index];
+        if (block !== e.target) {
+          block.scrollLeft = e.target.scrollLeft;
+        }
+      }
```

Comment:
`forEach` can simplify the code here.

Reply:
Use forEach will cause lint error `Assignment to property of function parameter`. I don't want to break lint rule here